### PR TITLE
[jit] Exported a method to get the supported list of elementwise ops

### PR DESCRIPTION
**Pull request by navahgar**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #59581 [nnc] Added micro-benchmark to show perf improvement with cat subgraph optimization
* #59580 [nnc]  Added a transformation to move users of aten::cat to its inputs in the NNC kernel subgraph.
* **#60162 [jit] Exported a method to get the supported list of elementwise ops**
* #59579 [jit] Made a list for element-wise ops.

Differential Revision: [D29190841](https://our.internmc.facebook.com/intern/diff/D29190841)

### CODE REVIEW

**Comment by navahgar** [comment]
@navahgar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29190841).

File: torch/csrc/jit/passes/tensorexpr_fuser.h [hunk 1]
```diff
@@ -60,6 +60,7 @@ TORCH_API bool usedOnlyInSize(Value* v);
 TORCH_API Value* broadcastSizes(at::ArrayRef<Value*> sizes, AliasDb* db);
 
 namespace tensorexpr {
+TORCH_API const OperatorSet& supported_eltwise_set();
 TORCH_API bool isSupported(Node* node);
```
Comment (bertmaher) [inline torch/csrc/jit/passes/tensorexpr_fuser.h:64]: While we're here let's fix the naming conventions so that we consistently use lowerCamelCase.

**Review by bertmaher — APPROVED** [review]

**Comment by navahgar** [comment]
@navahgar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29190841).

**Comment by navahgar** [comment]
@navahgar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29190841).

**Comment by navahgar** [comment]
@navahgar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29190841).

**Pull request closed without merging**